common/source/docs/common-polygon_fence.rst

```diff
@@ -0,0 +1,65 @@
+.. _polygon_fence:
```

Comment:
does this label need to be changed to "common-polygon_fence" to match the file name?

Reply:
fixed...didn't matter because only reference was via TOC entries which use the PAGE name apparently, not the reference label...